stac_pydantic/api/search.py

```diff
@@ -102,32 +98,43 @@ def validate_bbox(cls, v: BBox) -> BBox:
 
     @field_validator("datetime")
     @classmethod
-    def validate_datetime(cls, v: str) -> str:
-        if "/" in v:
-            values = v.split("/")
-        else:
-            # Single date is interpreted as end date
-            values = ["..", v]
-
-        dates: List[dt] = []
-        for value in values:
-            if value == ".." or value == "":
-                continue
-
-            dates.append(parse_rfc3339(value))
+    def validate_datetime(cls, value: str) -> str:
+        # Split on "/" and replace no value or ".." with None
+        values = [v if v and v != ".." else None for v in value.split("/")]
 
+        # If there are more than 2 dates, it's invalid
         if len(values) > 2:
             raise ValueError(
-                "Invalid datetime range, must match format (begin_date, end_date)"
+                "Invalid datetime range. Too many values. Must match format: {begin_date}/{end_date}"
             )
 
-        if not {"..", ""}.intersection(set(values)):
-            if dates[0] > dates[1]:
-                raise ValueError(
-                    "Invalid datetime range, must match format (begin_date, end_date)"
-                )
+        # If there is only one date, insert a None for the start date
```

Comment:
```suggestion
        # If there is only one date, insert a None for the end date
```

@eseglem ?

Reply:
oh wait maybe I don't remember well the spec 😅

Reply:
ahhhhh I see https://github.com/stac-utils/stac-pydantic/pull/131/files#diff-cedfbbe7e0d06171e4f3cdfab0c02e1712111610dc9419080077b4d15a1c3d85L46-L53